GoogleForCreators/web-stories-wp
# Video Cropping: Add post meta for original

## Context



## Summary

Save a id of the original video when a video is cropped. Add a new post meta key and related unit tests. 

## Relevant Technical Choices

It seems like meta data was not correctly saving. This PR seems the trim data, blurhash and base color are now saving when media is uploaded. 

## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a story. 
2. Add video to page. 
3. Trim video to half length.
4. Save story as draft. 
5. Re-load editor. 
6. Select video. 
7. Trim video. 
8. See full length video and retrim. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12310
